sdk/src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -348,6 +349,15 @@ public synchronized Client setOperatorWith(AccountId accountId, PublicKey public
         return this;
     }
 
+    public synchronized Client setSignonDemand(boolean signonDemand) {
+        this.signonDemand = signonDemand;
+        return this;
+    }
+
+    public boolean getSignonDemand() {
+        return signonDemand;
+    }
+
```

Comment:
Is there any reason why the user of the SDK would want to NOT sign-on-demand?  Is there a reason the user of the SDK has a say in this?  This seems like it should be an under-the-hood change.